conn_linux.go

```diff
@@ -463,11 +470,6 @@ func (s *sysSocket) Socket(family int) error {
 	// For a more thorough explanation, see similar work in the
 	// Go tree: func sysSocket in net/sock_cloexec.go, as well
 	// as the detailed comment in syscall/exec_unix.go.
-	//
-	// TODO(acln): update this to mirror net.sysSocket completely:
```

Comment:
After re-reading this just now, I should check out the upstream logic and probably just set this flag instead.

Reply:
And fixed.